Source/Core/EllipsoidGeodesic.js

```diff
@@ -9,6 +10,7 @@ define([
         './Ellipsoid',
```

Comment:
As Patrick said, please remove `'./DeveloperError',` and the corresponding variable `DeveloperError`. Thanks!